docs/recipes/structuring-reducers/UpdatingNormalizedData.md

```diff
@@ -195,80 +195,73 @@ This approach makes it very clear what's happening for the `"ADD_COMMENTS"` case
 
 ### Redux-ORM
 
-The [Redux-ORM](https://github.com/tommikaikkonen/redux-orm) library provides a very useful abstraction layer for managing normalized data in a Redux store. It allows you to declare Model classes and define relations between them. It can then generate the empty "tables" for your data types, act as a specialized selector tool for looking up the data, and perform immutable updates on that data.
+The [Redux-ORM](https://github.com/redux-orm/redux-orm) library provides a very useful abstraction layer for managing normalized data in a Redux store. It allows you to declare Model classes and define relations between them. It can then generate the empty "tables" for your data types, act as a specialized selector tool for looking up the data, and perform immutable updates on that data.
 
 There's a couple ways Redux-ORM can be used to perform updates. First, the Redux-ORM docs suggest defining reducer functions on each Model subclass, then including the auto-generated combined reducer function into your store:
 
 ```js
 // models.js
-import { Model, many, Schema } from 'redux-orm'
+import { Model, fk, attr, ORM } from 'redux-orm'
 
 export class Post extends Model {
   static get fields() {
     return {
-      // Define a many-sided relation - one Post can have many Comments,
-      // at a field named "comments"
-      comments: many('Comment')
+      id: attr(),
+      name: attr()
     }
   }
 
-  static reducer(state, action, Post) {
+  static reducer(action, Post, session) {
     switch (action.type) {
       case 'CREATE_POST': {
-        // Queue up the creation of a Post instance
         Post.create(action.payload)
         break
       }
-      case 'ADD_COMMENT': {
-        const { payload } = action
-        const { postId, commentId } = payload
-        // Queue up the addition of a relation between this Comment ID
-        // and this Post instance
-        Post.withId(postId).comments.add(commentId)
-        break
-      }
     }
-
-    // Redux-ORM will automatically apply queued updates after this returns
   }
 }
 Post.modelName = 'Post'
 
 export class Comment extends Model {
   static get fields() {
-    return {}
+    return {
+      id: attr(),
+      text: attr(),
+      // Define a foreign key relation - one Post can have many Comments,
+      // at a field named "comments"
+      postId: fk({
+        to: 'Comment',
```

Comment:
There is an error here that drove me nuts.
It should be 'Post' instead of 'Comment'.

Reply:
Yep, my bad! Opened a PR with a fix.